src/ui/components/Main/Keys/List/BottomBar/BottomBar.tsx

```diff
@@ -0,0 +1,11 @@
+import cn from './BottomBar.module.css';
+import { Button } from '../../../general/Button/Button.tsx';
+import addIcon from '../../../../../assets/addIcon.svg';
+
+export const BottomBar = ({ openModal }) => {
```

Comment:
In general, try to avoid use 'return' when it is not needed

```
export const BottomBar = ({ openModal }) => (
  <div className={cn.bottom}>
     <Button text="Import key" onClick={openModal} src={addIcon} style="secondary" />
  </div>
)
```